timemachine/cpp/src/neighborlist.cu

```diff
@@ -356,8 +364,19 @@ template <typename RealType> int Neighborlist<RealType>::Y() const {
 
 template <typename RealType> int Neighborlist<RealType>::num_row_blocks() const { return ceil_divide(NR_, TILE_SIZE); }
 
+// max_ixn_count determines the number of tile to atom interaction counts. For each tile that interacts with another
+// it can have TILE_SIZE interactions. Note that d_ixn_count_ is only the number of tile interactions, and differs by a factor of TILE_SIZE
 template <typename RealType> int Neighborlist<RealType>::max_ixn_count() const {
-    return num_column_blocks() * num_row_blocks() * WARP_SIZE;
+    // The maximum number of interactions is that of all tiles interacting with all other tiles. Which
+    // is the upper triangular matrix.
```

Comment:
nit, comment clarity: had trouble parsing these comments on first read. the word"interaction" is quite overloaded here (atom-atom, atom-tile, tile-tile), might be helpful to be more explicit. (e.g. saying "tile-tile" or "atom-tile" instead of just "tile")

Reply:
Tried to clarify in https://github.com/proteneer/timemachine/pull/1163/commits/f7634b031660b8376ec7e104aed3097c3328793a let me know if that helps

Reply:
Nice -- thanks!